src/components/Form.spec.jsx

```diff
@@ -1,15 +1,16 @@
-import {render, screen, waitFor, waitForElementToBeRemoved} from '@testing-library/react';
+import {act, render, screen, waitFor, waitForElementToBeRemoved} from '@testing-library/react';
```

Comment:
left-over unused `act` import

Reply:
I'm too used to having flake8 check this in python :sweat_smile: